src/cv.jl

```diff
@@ -22,9 +22,12 @@ last valid iteration.
 * `verbosity::Integer`: keyword argument that controls LightGBM's verbosity. `< 0` for fatal logs
     only, `0` includes warning logs, `1` includes info logs, and `> 1` includes debug logs.
 """
-function cv(estimator::LGBMEstimator, X::Matrix{TX}, y::Vector{Ty}, splits;
-                               verbosity::Integer = 1, truncate_booster::Bool = true) where {TX<:Real,Ty<:Real}
-    ds_parameters = stringifyparams(estimator, DATASETPARAMS)
+function cv(
+    estimator::LGBMEstimator, X::Matrix{TX}, y::Vector{Ty}, splits;
+    verbosity::Integer = 1, truncate_booster::Bool = true,
+) where {TX<:Real,Ty<:Real}
+
+    ds_parameters = stringifyparams(estimator; verbosity=verbosity)
```

Comment:
I think this is a wiser approach as it reduces the chances of missing or wrongly allocating params and hopefully is futureproof